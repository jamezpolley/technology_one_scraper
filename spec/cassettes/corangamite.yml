---
http_interactions:
- request:
    method: get
    uri: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationSearchResults.aspx?Field=S&Period=L28&f=$P1.ETR.SEARCH.SL28&r=CSC.WEBGUEST
    body:
      encoding: US-ASCII
      string: ''
    headers:
      Accept-Encoding:
      - gzip,deflate,identity
      Accept:
      - "*/*"
      User-Agent:
      - Mechanize/2.7.6 Ruby/2.5.1p57 (http://github.com/sparklemotion/mechanize/)
      Accept-Charset:
      - ISO-8859-1,utf-8;q=0.7,*;q=0.7
      Accept-Language:
      - en-us,en;q=0.5
      Host:
      - eservices.corangamite.vic.gov.au
      Connection:
      - keep-alive
      Keep-Alive:
      - '300'
  response:
    status:
      code: 200
      message: OK
    headers:
      Cache-Control:
      - no-cache, no-store
      Pragma:
      - no-cache
      Content-Type:
      - text/html; charset=utf-8
      Content-Encoding:
      - gzip
      Expires:
      - "-1"
      Vary:
      - Accept-Encoding
      Server:
      - Microsoft-IIS/10.0
      Set-Cookie:
      - ASP.NET_SessionId=fa5yrb55b1g2w1zm2o1llr2m; path=/; HttpOnly
      X-Aspnet-Version:
      - 2.0.50727
      X-Powered-By:
      - ARR/3.0
      Date:
      - Tue, 23 Jul 2019 00:17:08 GMT
      Content-Length:
      - '8117'
    body:
      encoding: ASCII-8BIT
      string: !binary |-
        H4sIAAAAAAAEAO29B2AcSZYlJi9tynt/SvVK1+B0oQiAYBMk2JBAEOzBiM3mkuwdaUcjKasqgcplVmVdZhZAzO2dvPfee++999577733ujudTif33/8/XGZkAWz2zkrayZ4hgKrIHz9+fB8/In7j5DdOHv+uT788efP7vDxN5+2iTF9+9eT52Un60fbdu9+9d3L37tM3T9Pf+9tvvnie7o530tdtXUzbu3dPX3yUfjRv29Wju3evrq7GV/fGVX1x982ru+8AZRev6a/bDb8znrWzj464Q3ycvluUy+azCIzdhw8fyqto/nieZ7Ojx23Rljn9+WMnVZ0tL7JF0ebp63lR52n+Oq8vi2nepNvp6/WEvmnzWfo8a9p07yB9ml03BOWuAHhcFsu36bzOz6Xnhro+r5ZtM76oqosyz1ZFM55Wi7vTpvk9zqmX8vqz19W6nuafvM6WzScv6+qjtM7Lzz5q2usyb+Z53n6U3v1acL9Yl8U3BuwnMwKU3QhuPL5L/zterX7/N/N8kTd3T16f3H2an2frsh0TuI/S9nqVf/ZRm79rAT4O8C7PCVF1Us2u8bOZ1sWq9d/96ewyk08/Spt6+tlHd+9mP529644An90ti0lz96d/0Tqvr+/ujg/G9/SP8U83H1FnAuYo/Y2TlJ7H51W9SJfZgnrKmtUyb5/RBx+li7ydV7PPPlpVDXWZTduiWn72Uf6mzqZvabxlMc3w0es8q6fzV3lDA27GBODd7/GsyMvZZ69/YbZYHb7M64KgPN874D/PP/vd9/Zf7o5P37wavz49fnXy7fFr8139GdFu/N3TJ59/dfr6zUdpQZ0TPIMQyDIrLvGjWK7WhjjzYjbLlx/pAH7/3/8nz06/+/rN8ZtTARB8cJmVa2p09+r05dOrr36vL96c3v/i6cUPvnj6dufF1cO3zy6+s3i69xPfPb44e4qfT47PTvXnE/w8Xvw+L3/qye9zenxxip8nP3Wh71wdENb7pz/5+U/uz46/+s6r775azU6evJjsvSq/fdE+nezttr/Pd++/fbN41v7UT3xF7Z79IHu3//In9h62k+8+W//U6bPrn7r3k+1Pfff+zpNXq5/44quf/Imv3r7Ynd57dTn5bvmD3+f3frWa7N1/9lOflztvvvvw7U+92v/OT3z3xU//1O/94gev937yPv39Ez+x92x3uveTP/jJxbMm++6r+ezz8nKyuMhPnr2a/z5vy/VPEW7HP3F8/O2L2VeTvYfNT35eXtF7L4P2b2fXk3s/efVT3331lmgQGePB8ZOf2H39U9/9qWvquzo7fTH/ffbm5bevTp89oX6me8358cVX+VdPnz376tl3ns5+7xc7NPbVdO/ZTvbdh+uvFj+5mC5+8geZfe8F/f9V9e2rr54cHx+fP1l8df5k7yff/tQx0Z/6O57vf/snF+UPsu9+p/mp784Jj7OT44uDlz91fPDy2UlzfPzq9/n82xevvvx9vrtbPvmJVTPd+andyeIF9/Xti3enhOd89t13Oxjj7LtfPTu9fvUTX7y+/+wnn52tv9r5ySdE47Pn5Ysvvvjpi/xp+WQ+xfz83q/KaflT88m3f7L8qSc/UR//xOnJ8R7oMQU9PsfcP333+5x8++r4WXtxdvATP33/JyafPyMtSTS+fvITP/V7f6fNfu+fuPhq7ycJHtHo2XfK6b2fbGbf/uLg+c7ZJ2fHO7/Xy9PvvH55sfN7vf7u73ORf/fh7tnn8/ls8dUF8cj9s28/2yXa8him10+uf+q7M4JD87+YXsw+n5dnn//U5eTzd5ezPe5vPv32E5q/FzJ/yy8upp+/K3+f3/sFtXtBf9Pc3/sJee/Zk4bgr7Pv3l+endIYP392Pft8t5wsf+Jisrd/8cXTL3ZfvHn7gxc/OL364mT/4s29V9VP/d5nF7/P3sP17PNnP038/JZoM8++++4HBHs+KZ4UP/X6yYL4Zf1TJ8cHP/H6SUVzu3j5+jvV7NuvrqY/qC6f35vdm13f/+nJve/MJ4vZnHh556de39/LvvvF+qf2Hu49Xzzbfb77hHjuSTldvrpPv1scn+++2CG+3519/vA6f73r0fghydP9nd/nuy92nn/3tP3q83cE+345Lc4+yT7/yeufenN1+RNv9g9+ojz75Kc+/4mTY5Zr+kl8Q/xzfnz81Rd2bk7B9z/5A2lnv/99vvjp3+fyi6f0/58+fvfl6yc7k+vj6y+uD65eXB9cf/H09D7aP/kJtP+J/OR0dk3y81X+ez8hHnxHvPWdJz+x07559dWz1z+xc5GfyvdPpwsa++/9ndXv89W71fTeT1w/+AkSRvrf8cu79ByA//W5OqF/jr969hPvnpfH9P9Z+fsU91/+PotV+fvce/Xkpz5/dkU4X0+vry5+ckE/92jef3rnHc3ZzvOfPl5/cXJ18RPMb6DF7vqnfu+f3CG+bp4TxjOSI+KV5z/1e5ekA9py8tO7a5KP5snxKfV4Mj3+yRf3SeeUk9f3icbvSOYMH97/Nummksbw08/fAv9XF1/8ZDv5yaen66/o/z+595PF87cPi2zxkz89O3329vf5vZ9grD94TvokGPtrwpnab3zn5Ml38RP9vnxzevX8p3+CxrR/RZ8/JVx3Zr/3d8qXhMvs9351/ft896ohvt79fRbvVr/PDnkjP/nqMtv7yTV9v0syMvu9f+ILGtfx57/3HvX/3d0fPPnJhz/Ifu9VefLs4Z7pA4T/zu5NtL7/bRrD25+4950V/s6++2zye//EGb15TArp7VP6KXPJdDz+9vEV/Xt8DNDyz/FPlDeOu0srgf8deX/n6RPwy/Hx14DzFb12/O3J7/3iu7/Pd9/t/tRPzNvprui3Ly7ut9Odh1fUjuTg4TXZo7c/9RMr+ox0x7dpgt+Wb58udn/w1be/czmlcc++XbKuJZ34hP7PfPrkguF/N5SZ43df/HR19cVT+v/JkydvfoL0Jj3PfvrsB89/cHzv+Q/Orr548/biizenn5LOwf8vvjrd+fwJ0+07pz9x8fvw4J9+9e7ypz6flZPv/uR6xrL61Xe40+OWZPo7bKutbb6gn2wzYKNKnq+zZ6/uE96ePTk4yz7vtH317svJ9RPSrz91OV3sUh8kT8WTFele0snPVqR3isnnX5H9/Inf5/jqJ58SXzRfPD1+S7QivUh4nbz6abJzr0ln706+/eoHr4hHfurkFdnHJ7/3T333O4zHdz4nO/j0+Oo7n8+uf+oZ8ZjM0Ve/z3e/Q3z67JpsXDm7Ojj4va6Of6/jn7g4Of7uTzz7CfPOKc3FkmRy+RNvf597P/kDsqE/gBxMdwDjJ96Szr0CL2i/Pz37/Ipo/urt7Ns/Zft/83n5g9mzn1r91O89Zbv5hOzZT33+imh4fPL085949u1IX5PPv9MYPfnq82fES1+8zD5//Z1Vtv/d87PievdgvayunrxZ3Xv2+3zRrr7zEz/x2WdwZck7uyteGv1mfvmx23lsn5++OH11/ObLVz3XzftGfbgHD/ef7J7uPpM+N3dw+pOnL9785PHzs6fHb86+pDhLoPc+VtDkHn732fHz64P693pxfPF73bv/8Nu/19Or5/cOPnm98+Qnnj/d+am7d0+On6/rdnn/5Pj3evrTqx/k9Pfd4hd9uXx69Xvlxaf8c/enM/x8PpW/nz+Qn79X9tO/F/+cS/vnZ4X8ff+nn58XTy+eXxQ1/7wrPwneA/559+Lgu1+c/MTv9dOL75y/Oln/ovletn+6ePjTv8/b6bc//+knxYO3P3j9bn92ciYkkWkQFx8PfULTkU7LrKEgcUJO/EVdrZez7SkFQlmxzGt42GhHje3jv4MYJa+3q3VL/wZv/Rg9kZa2TaoflNk1vb29l1JDChvOi3fm7djrDUWk22V1UdlGP/Y407DLRHGIcqcugB1T2EoR0eU4W9/13rqbmd8NVehX/3f/V/rNILLMLosLjnI2jhrsRE0ptKtK4h/6Vd/6KALIgUhnWZttL/Llelv5tqqLH+BrjtMJuoBXGBd1weE+fRx8Pq3K7XfN9u6e+fLHHq9L82Vzzh0IyzfT7QV1LZ+Yxj9G0axpTWhuNzlHetu/aJ3VNGQCTGxwXtRNu00ERqB4yb9Qw1+0vevAvNfcfLta5D5xtsv8Mi9DcBYrenexXRDdELsWR4+bVbYMvkN8/NERYFJsS196QNzM0+9l4f64xaDxBcPHSB116eX3GOkJydi0KGODTT860m+/Npp7jCRi88t8e5W18/Dvi5z/JpRkEPe+5iAoSbPK6/Z6YBTm6w8YRofa+18T0ZNqsVgvi0FM7fdfG9V7XVTvf01UX5YZZdi2M9LAhB7logZQlnYptUvpM2r3zaH+6ddE/cm6IQWGXJbXnWL80ZH59mvjud/F88HXxPMEunRKySuvN4umfpl+NYjo47vrUn/37MPQ7+ZX/EcDME+oqZctzWDMllDL4OHXoLOhsI1hkd8VGP7oQqLvnFklKL2HwSoAvAVsGBD+oJ9xZMwTvt21Oj9GLYImDLhYEtAQPDXFQ63tw6+Z4XJrwiwcKj6kn7cZ5O+6vZ0+ry6KZbq9PdQGHSr4ZpVPi6xEUncIJJ7hb/AEVok73wTLPMTNGPW0LXd2fn/6l1/8/aeLGf+CrCvJfvM2vyaYlElGxv2zj475w/T3yq8fpfSpiINLDz/6/X//WfWSkrZPyMfb+pig7uz8bvQvg/zdDOyPRx9/fOejI/4dAkDIbHxuGIs1vPT74DPwrZEc+rX38Fyeku5jRDGf9KF9+u/wvDJJm+ZJBt6jF9mBpC97D5FfqGeUSd7oyseQSnmz+/LVqq5md7+bTygJ3tzNjfG7+3L37ufrvGnhh0gePExo/8Lzz343ZL5fvhrz39/+4nT8k2ffFdeM5gccsJjh7RfkSw5gjOdxsbiQJQBaeaD/ER5m8UEXHu6eLcj2N3dPF6v2enxRnPudFPjOdJNOqpp87c8+2rHSNjdfIYqg7noP2KX/TWwWo5/RR6YrlmR4dfRZrBkj3TRQ1aQPjus8i9HFb/oma94SI5JzKX2kn6X4+EX1si5Io0TexhP/FHAHXsAz/I15AEDQIB1GPvdLQoxde/pu8Nn8LZ4u2G9zuHQTXPOIomJyMTsYiv3+9ar+gqA9L5oWqmhn9/cvJyU++ZzixJXp5Msl6ck8fZldU9dwRgDuFl3H5jj2eLELdSB9AyUMD57DkZVZ4vyc0Lh7vCSOLuk3at9GBQ+LS3bh6eXvMz5+cfbF8fPX49MXP/HRkbxuBgTeZh8g3pVb9qI/0f52/b18aTpzAG7X49N80lY1/Yamt+rs6ekTypNof/L67bo6W56TjFyQHlm29Dd+3KrDsxfPXp29+PxUevSh3K7fV1mbN/QLWt6qQ0oEner4+N3bdfNarDy/QX/jhVv19vrl6Ql6lA4VSspg2Ck/mWc16VqDxCASr+fVakWUOSEvN/jjRixeHv8+45PjV2/YWhyZV1O8a3uj7I54qyRCG5/Hk3pQsZtn87d4fkgqaC+qgr4iI50Sn9GiMEvSz60OAjZ3X1LjK7KktAye3zyhcAG+en366uXx69cyqbTAfVG1ZOSIjQSSndlof6/yC8Inr2/V1avTz9Hb+IsXb0hi9E0L/4fNObeYg+Fv4998PZhDb/U/YW5ndnUOxqu8JBUwe05BA8XByi5f39fA47oh1t/5/V8VF/MW3cFt+P3bZhV2KT1+1GoD9isYgveFkRh9M+VXzbiJy/EvsYBlr/xNTTGDZ6Rek382nRNXk1PZCLM9K/Jy9tlr5rKXeV1Us8/efJf/2sSGb16NX58evzr59vj1G+L317SWWxC/z9I386JJv5vnb32GpH8/FKfn74XT8wCn51nT/mzg9OYL/uuWOL35wseJ6fQFOcLzbxip5++F1PMAKSbUzwpSD/iv2yJFCaIOUg/Sp9k1sfo3i9XuPv95W7R2KY3ZwWt3/2cFsb0D/vO2iO0d9BDbO/iGEHvK3Spi76UZnkIzPM3JOC0oxFCW/2bEMEDqvVTDU6gGDykm1s8CUu+lG55CN3hIMaW+ITkMsHov5fAUysHDikn1s4LV+2iHp9AOXay+KfUQovVe6uEpNe8j9o3phxCz99IPT6l5H7OIghC/set0PKsqevOjo1+4nDSrQ+tt6M9h9yf+zft8ypiwA9U0387L1fFydjyjIcA1oq+jj3xj/pWf3cfgTr8Gz9Dn3affwsdUs1sxJLmZkta1/G5Vv6WIZ5rHXsFD7BLCflOtkGWLpe3wxD/F83i+K6DYIxV38vef776kDOIbJKOHzch89706e59P3RP/Xj6Vf7uhphLk95+um9ONsvRR2rTXyLbPimZVZtePCk58bU/Kavr28KqYtfNHuzs7v/shEfYJFumWSwTlJHBEjlRApQrr8d0nR4TP4yeviCZn5+l1tU7n2WVO+YPr9BchZ0y9N2mdX2T1DFDaeZ6eV2VZXeGvzKHYpCtKmTZ5yqshtHqFlrbzp/mKcgIUsrYp5Zd27t2///DeA0JynH5J7WjhRF+a5bSSUjbplEgzQU9rymM8PlYdY3LhGxbWXM5bu777us3aNaWarrftRyA19badyUdQCYQELdEfE8VeETtiaoggNEtBbCn/Dk0cJQSEuE8poqFZUungaaGpGH6L1eKLbJG/3CVdF4NxjhbPswliKfmeeJkaKKY3wf7JrFzfDvjep3d3Pr27t7P7MG2rdO/e3Z0H/JfpSH4IHeRfVgT9romLlcU+r4vZ779alifrpq0WRcPcgg+hI35MgPT/Nc/twNf5ilY9+FdpR/29lsVbaAaac+5Me/OfIdLd1AGRttOBIacuGpuPbxRAIEDIES4bH58P/YeXXN4ff3pL8X8N0cxIQLoj8L6IL+mQxY4s52Rl+9lH9IroKFnE2RattLN6dzjPkTzgX+XD+9BUvaHRbFkbRv/znq4pBIdQK1rmbrNJSfpHBnFBY6UR5WVJBJ4S/WUhif5eZTPMzGcf3QvWmN6fhhf1TD94k03WZVb/ZJFf2ZHL6EjH/e6H0gutE5dltmryR+YX0tBAnDAnDShoz4lt8vpVdaVf0XfztJmSUiPsKpJQsqCiDG9YVtVB/G7hIH43NwgsvFK76CBoCfZ1Vbe/26tscTbDYizNulH0MNaP77bzo59TzJ5XM8nkQ5cBQ/5Ac5M/18hRzpTyv+R5HM9mNS2IAz/7Waof/r8D1ae59EATCyy9Py1+zIf0W02YWj5dYjhlwKezHw7m/N0e/7tLowDWL1/CRpGt+lSRJlTo//yRmi/6Ex/tfbq3m55Uk1m+3H5J46elisWKLB8tm1TZLH355as36cnxFy+fnD5/nv7k2Ul6b+9T9zYluMkxmpE3c5mX1Qrsl1bn7Ok8x+eU9+c/Xq7rVUVt6bss5Rx9ejydVotFNWMRSp+sixI6iCEz/ei3gL6kRfN6Sbz9c0bie/xvj8T3byLxg09301fV2/yqqmbbr98SgjRgpu7r3+vs5ZsvXwhd7326a18x5JBlKw4i0uOmqaYFEWCWXhUkIVn6k/m8oGX59KSu6LtLLBTQu0wa+i2g3s8xd+7zvz3S3QtJd0AfhaTb20lf5Ffsj4Gh8vTF6XefnZ0+fyok2/v0wDaNkQzsR8g3bb1mAy7890U2nVN4UF+nr+dEzK2spQ9f5bP1lP56nbcpxn6HATOt6LeAlj/3nHif/+2Rc+8mcu7fS59URJ3XbZ3TON+cffHkS8N+X4+Wr8kbIn+HKcmvMznot4BiP8fc9yn/2yPXfkiuB/RRSK5Ptw9oyXgN51QpBk14+urpl9+1RNuxrWlxmdbSQRZoxfO6WhC9Fou8JqEtETwQ9gWp2Zb+5JeYFPRbQKqfe+Z6wP/2qEXKaTO19iif8EW2LCgYthy2iV5DTAZ7QdIYWhQKSU7f0WIotU+/TTmrckQ2Z1GVBbMiAJy+Y8KV8Cb4U4F7DHqyjSH/gnplitJvILol+s8xfx7wvz2KE6l8ilMg2qH4l0Toz3MiExHF2JMnX96O0l1r8lVbEDGvaa2+aTNyxkGxdAvOQF7Pqiuy0EXVEL1nTfqmzjPJXDyjOAJv3WHAr2hCLmkCaLpe0PuUM/nJ/ILSFwDFuDBt6beA9j/3DP+Q/+2SH17OZvJ/StbplJhteWENOqnT3+v4+FV/CqASQKOOo0Q2TZykLH16RU4XwUq3nmX1gmcpr5uPQz9pg1X6ueXh3R382yci2RSPiPd7NNzdfXA/fVa9m1dXQsHnp6+fn52mXxy/+DJCxhgnk2aF7NPfVkOckta9yJdTsvB5TbmwvLGsmm49z5uyyKGqiO7PipqMl/DoBuL+nDPp7i7+7dP3QUDf/R599x7spL9XBYGek3u/oLh7mVPKhUn98ksy/69enJ696RP6PfgVH325bic6Nfy3pwpuowNoJn4ueTceO30axk57ZPEkEWg/ur+zkz4tLpb5dZM+Ibhk/Zmwg37Vm6sqfV5Rxms9oRxN0RBR0q3Td1P5jYhliPr/alaMx0GfhnFQhFz7u+RZF2VOUinewSCdYmLeNVgZculT9rfoq3wxKUm23xTE4kTKL3Iy//BJ76TbKUUQxIWUHJ9ltLpJ4Jlo9FvNv9EvSJPhd5tg857wr1jS8XZ/IUtn4Lv/6Av6hv7hrJ2dY/awn6xbMizN8/w8WHB6XCxXa1pNuEZ+pOElnY/SZbagvzAvboYn7fJlnV8W1ZpS2ySNa2rhPuhn94L22ZS0ZvM2v6Z3PkpbLCB99hHZIvow/b3y60cpfarYAsHjuq6uBGGXuvSHS3/2Hn/IFMdmtOpgXqFve0+M9ub5Ot8N9RX7PPrZitBd9T40IyqWkHDiLIyq14zAyRQ0zZOKyLZ4QsM3r9KHL6qXdbFhpbH77nNakKJUGX0dPv3X8ZCyYAA8//OVLkXIavDzYvl2N2CAn4wyAH0qCucWi1Cf5xRvZ+U2FiMbWpIqLrPp9TbsLqcsCX5GBpvS5L9/k5fnpFSkBTTKwAAoXe8Ngf4S5G2KnkagxKS8SEuZ2pKIRKnom/L2pChM6yekX97GUviasd91Kftd+o1sKyWqvcS+E4TucxP596hHpjST/w2Rp09++vTW5H9NbnyZK/WfFg1RpiAXaZtU4vZJtbquMR7qJZwE1/AD5oGGQkD+PzoPtB7izcNXRKH+PNCnt54HaFrS7QQnpLR+TvmC9ZLcVJCbsIs9A+OgFR1/ze6LakJ2FiOI6Q/zdOaOCf//arHB050yGSp8EzMPN/g/wTvqw8gHKT55f9oPrVfGTAaeqPp2Unj0C2nl4/owfZNP58uqrC6u0y8p8fm8ADPNUvGoCEQHcuwz95f/LazxYyGQOhFt/q6968gGnrl79/Hv+r2Tp8dvjr/3Gye/29asmq6hqO+MKfafXW+dE5/CRUm37qS/uDhPt363rY9/nOn7+5OjcUrOAIWud8ZFs/XxI7WCH99JP/sspexhTq+g+e//+78+efXl8+cvv3x99ubsyxe/N71AL299vPPxncNYg9/Hb/BLfsmdw7t3v/99wpVmgPHmXyk4WdAv6srR+ja5enjkb/Nn8Jf/h/ndUoxnS0XinBXDdrXGv1PIbEFmjeIh/vi8qJt2m9wg0I/oHDwRMA4A+z7nxbvYi3j8l7GKm+KfbVq1HXoBj/8SraJtv2u2d/dS/NZsf8o/F9v7Kf1CWEzb7XWzrXiRJJJuAmiiB57H83sGkDZhNejU1leUWpvf816g9WQs6FEDqwbJG6dQ27hi1AjP7sEu4nA/Zdhp4RJQIxLPKRmUIhshWKBQ1fRiG8MhM1GQr40vquqizEkpL6CEF7TQfHdFSiq/S/1/Yvr/5HU7+sR19wkFJZ+gn7u/5/a9g/Hevb2DT3fuj3b374139/f39vZGuw9+cJeyMtlnv+u9xe7vuj+h/y/u0+/7v+tus/Pu02y2l+e7k4Pzg/vTT+/ff7Tz7sFsZzaZPTyY3Lt3b292797verDY+13vzTzwv+v+TDr4dH9358HvMS8/y8mdtsZiUmbQ6FBTKcneF9mKiHDUoVjnzzd5ma/mFWmQR+nWzr076f37D++lD2gVvtPwdEGxGTWyFMTfbfWowbz9nnGD9tHRpm8ZOfZ8CTw9RqToV3o28af+Aga9DR++nmeUxfmFNHeHZEKXy3zadhhyXXbepCCbYkoEktsFiUCTcnBJXeazIpNv4UxbCD/2uCwiILbPiY0o4qRZcU2J/5WEPhOalmBDcgQswVguvEkWj8CHVhAVi6Pnxds8XTeYeAPKb0S0dn8+vlsW/p9x3FsaPv0RdtZFXRv1sD7xUFa+9MEIzs+qsqTEnmCtkPxWXwPpAvnpizpbhP110QbFbVNG3uNRZtq73gCGaM7JcEDwv/oaSF+TyVhPiFNdyzjK2pARXjd53cP6Fkh/N2tpwapa1+nvU63fEDRaw8pIKEq/aX8Qjtcf312X9lsjtGRRes/Qd+Zz+5N+dMhC46R/SfLUBOrHTU4fzX64RjTsO6uvec5I/ONw8PiwoKic7lpsP9j0Ih68HLdT3mR7arQTPLuorBu+dVgj9QI4zqPZppj9X7icNKvDn3Hq/r0R0Rj95iieLLu2+GY6fk3fkhHvdaeff91Ojrd/avtsOcvf9QAfUxLxp2hljL4DcMPWNJfR58bv6esh7rmfrqqrvM5n25Pr2zCSgcPzetMLeDS6oDDiATGEJYS4Z/QJOfZFqeSTHxJcaj8+di/l93RyHdJZyVytaAWGVmLzhigOz8ujq1qML6nJCTcRuqKnG8ZwI3E3fD/0nfnc/KSP8LgP8Nukml3Tr+RYtAuox984+X8AHUm7ED9eAAA=
    http_version: 
  recorded_at: Tue, 14 May 2019 14:00:00 GMT
recorded_with: VCR 5.0.0
