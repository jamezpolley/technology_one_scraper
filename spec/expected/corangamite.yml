---
- council_reference: PP2019/065
  address: 41 Bailey Street TIMBOON VIC 3268
  description: Buildings and Works Associated with a Place of Assembly (Timboon Mens
    Shed) - New Verandah
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F065&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-06-27'
- council_reference: PP2019/066
  address: 500 Digneys Bridge Road TIMBOON VIC 3268
  description: Two Lot Subdivision (Excision of Dwelling)
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F066&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-06-27'
- council_reference: PP2019/067
  address: 270 Koallah-Pomborneit Road POMBORNEIT VIC 3260
  description: Use and Development of Land for a Dwelling and Outbuilding and Removal
    of Vegetation
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F067&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-04'
- council_reference: PP2019/068
  address: 1175 Foxhow Road LESLIE MANOR VIC 3260
  description: Buildings and Works to Extend Existing Emergency Services Facility
    (Leslie Manor Fire Station)
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F068&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-05'
- council_reference: PP2019/069
  address: 620 Eddington Road BOOKAAR VIC 3260
  description: Use and Development of Land for a Dwelling (Farm Workers' Accommodation)
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F069&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-16'
- council_reference: PP2019/070
  address: Old Geelong Road SKIBO VIC 3260
  description: Buildings and Works Associated with Utility Installation (Camperdown
    Biosolids Treatment Facility) and Removal of Native Vegetation
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F070&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-16'
- council_reference: PP2019/071
  address: 228 Manifold Street CAMPERDOWN VIC 3260
  description: Buildings and Works for the Redevelopment of an Existing Hotel, Demolition
    and External Additions and Alterations
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F071&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-17'
- council_reference: PP2019/072
  address: 43 Bond Street TIMBOON VIC 3268
  description: Buildings and Works for Construction of a Storage Shed
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F072&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-18'
- council_reference: PP2019/073
  address: 20 Newfield Lane NEWFIELD VIC 3268
  description: Buildings and Works for Construction of a Machinery Shed (at a Reduced
    Set Back)
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F073&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-18'
- council_reference: PP2019/074
  address: 6-8 Church Street CAMPERDOWN VIC 3260
  description: Change of Use from Commercial to Residential
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F074&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-17'
- council_reference: PP2019/075
  address: 761 Rokewood-Skipton Road SKIPTON VIC 3361
  description: Buildings and Works Associated with a Vehicle Crossover
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F075&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-19'
- council_reference: PP2019/076
  address: 2621 Cobden-Port Campbell Road PORT CAMPBELL VIC 3269
  description: Use and Development of the Land for the Purpose of a Group Accommodation
    Building
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2019%2F076&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2019-07-19'
